AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/utils/AppUtils.java

```diff
@@ -51,11 +57,13 @@ public static int getResIdByPackageName(String pkgName, Class<?> className) {
   }
 
   /**
-   * @param context
+   * get app's Name by packageName 1. Use packageManager 2. use built-in * resources(currently
+   * support 56 apps). 3. if 1 and 2 cannot find, return "*unknown" (* is to * make user it will be
```

Comment:
nit: "* is to make " ? extra *

Reply:
Oh, I didn't check this comment after formatting. Fixed now